How-to-Install-SCF.md

```diff
@@ -207,8 +216,11 @@ To install SCF
   To invoke the tests run the command
    ```
    kubectl create \
-      --namespace="${NAMESPACE}" \
+      --namespace=scf \
       --filename="kube/cf/bosh-task/smoke-tests.yml"
```

Comment:
Don't we need things like overriding variables that are meant for vagrant (e.g. change `DOMAIN`)?